samples/net/zoap_server/prj.conf

```diff
@@ -9,3 +9,4 @@ CONFIG_TEST_RANDOM_GENERATOR=y
 CONFIG_ZOAP=y
 CONFIG_NET_APP_SETTINGS=y
 CONFIG_NET_APP_MY_IPV6_ADDR="2001:db8::1"
+CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=5
```

Comment:
I'm with @jukkar here. 3 should be a "normal" value for "pretty good" IPv6 coverage (definitely enough for 6lowpan devices). Setting the default as 5 seems like waste of memory. Perhaps, we could try "default 5 if NET_COAP", though I yet to confirm that in such case, user still can override such value.

Overall, we have other case where "not good enough" defaults are used, we'd better fix those first before trying things risky on RAM usage in 90% of cases.